DNN Platform/Library/Security/Permissions/Controls/PermissionsGrid.cs

```diff
@@ -1113,6 +1113,11 @@ private void FillSelectRoleComboBox(int selectedRoleGroupId)
                 groupRoles.Add(new RoleInfo { RoleID = this.AllUsersRoleId, RoleName = Globals.glbRoleAllUsersName });
             }
 
+            foreach (var role in groupRoles)
+            {
+                role.RoleName = Localization.LocalizeRole(role.RoleName);
+            }
+
             foreach (var role in groupRoles.OrderBy(r => r.RoleName))
             {
                 this.cboSelectRole.Items.Add(new ListItem(role.RoleName, role.RoleID.ToString(CultureInfo.InvariantCulture)));

```

Comment:
Not a big deal, but since we're already iterating over `groupRoles` here, I would just localize it here instead of updating the collection above

```suggestion
                var roleName = Localization.LocalizeRole(role.RoleName);
                this.cboSelectRole.Items.Add(new ListItem(roleName, role.RoleID.ToString(CultureInfo.InvariantCulture)));
```

Reply:
The problem with this is that you end up with a (potentially) wrongly sorted list. You need to localize before sorting.